packages/starlight/components.ts

```diff
@@ -4,3 +4,4 @@ export { default as Icon } from './user-components/Icon.astro';
 export { default as Tabs } from './user-components/Tabs.astro';
 export { default as TabItem } from './user-components/TabItem.astro';
 export { default as LinkCard } from './user-components/LinkCard.astro';
+export { Code } from 'astro-expressive-code/components';
```

Comment:
Just to double check, I think you said this only adds CSS/JS when actually used right (same way the standard code block feature does)? Only because as noted in #1272, this module doesn’t currently tree shake. I remember you saying you had it set up that way, but wanted to double check.